hail/python/hail/expr/types.py

```diff
@@ -653,9 +653,16 @@ def _convert_from_json(self, x):
 
     def _convert_to_json(self, x):
         data = x.reshape(x.size).tolist()
+
+        strides = []
+        running_shape_product = x.itemsize
```

Comment:
This is sort of like `axis_one_step_byte_size`? Like, if I want to increment this axis' index by one, I move this many bytes?

Reply:
Yes, and that's a better name